config/environments/production.rb

```diff
@@ -77,3 +78,8 @@
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
 end
+
+Rails.application.config.middleware.use ExceptionNotification::Rack,
+  :slack => {
```

Comment:
This does that already. If you look at the initial PR comment, the channel (within Turing) is #lookingfor-errors.